docker-start.sh

```diff
@@ -15,6 +15,9 @@ else
   chia start farmer
 fi
 
+trap "chia stop all -d; exit 0" SIGINT SIGKILL SIGTERM
```

Comment:
```suggestion
trap "chia stop all -d; exit 0" SIGINT SIGTERM
```

Failing lint - can't catch this

Reply:
thanks for commenting on that. The lint output that I saw didn't give much of a hint as to why it failed.